ssl/datadog_checks/ssl/data/conf.yaml.example

```diff
@@ -0,0 +1,26 @@
+init_config:
+
+instances:
```

Comment:
Let's make sure we format this properly. See https://github.com/DataDog/integrations-core/blob/master/docs/dev/new_check_howto.md#configuration